test/test_base.py

```diff
@@ -47,7 +51,10 @@ class TestFloat8Tensor(unittest.TestCase):
     def test_preserves_dtype(self) -> None:
         # hp means high precision, lp means low precision
         hp_dtypes = (torch.float32, torch.float16, torch.bfloat16)
-        lp_dtypes = (torch.float8_e4m3fn, torch.float8_e5m2)
+        fp8_dtypes = (
+            FP8Dtypes()
```

Comment:
all dtypes would be nice, there should not be anything in `Float8Tensor` which is hardware dependent